lib/mergometer/reports/pr_trend_report.rb

```diff
@@ -40,13 +40,6 @@ def eligible_prs
             end
           end
         end
-
-        def filter
-          [
-            "#{repo_query} type:pr created:#{52.weeks.ago.to_date}..#{26.weeks.ago.to_date}",
-            "#{repo_query} type:pr created:#{26.weeks.ago.to_date}..#{1.week.ago.to_date}"
-          ]
```

Comment:
I think we need these, the GitHub API provides a limit for 1000 PRs per request, so this is a way to chop them up

Reply:
But all reports need PRs, so having a from to get PRs is okay? how else do we decide how many PRs to fetch?

Reply:
For clearing more, I will remove from parameter from all reports, then reports will just take PRs are parameter, and generate report for it, so it will be upto user to provide whichever PRs he wants.

Reply:
Then we will have a different baseReport class then now, see PR #8 , and current reports class will serve as a aggregator of reports.

Reply:
> But all reports need PRs, so having a from to get PRs is okay? how else do we decide how many PRs to fetch?

Hehe, all reports may need PRs (for now! 😅), but not all reports will necessarily need a `from` 😉 

I think adding the capability of having different reports accepting different arguments from the command line will give us the most flexibility?

Reply:
@balvig